utils/build_swift/tests/test_driver_arguments.py

```diff
@@ -220,6 +227,8 @@ def generate_option_test(cls, option):
             return cls._generate_append_option_test(option)
         elif option.__class__ is expected_options.ChoicesOption:
             return cls._generate_choices_option_test(option)
+        elif option.__class__ is expected_options.HelpOption:
+            return cls._generate_help_option_test(option)
```

Comment:
This can be done cleaner via a dictionary.

Reply:
When you say 'via a dictionary' do you mean I should have a dictionary from class to function? I'm not sure I'm following along.